ComponentLibWithXmlLinkerConfig/ComponentLibWithXmlLinkerConfig.csproj

```diff
@@ -5,6 +5,16 @@
     <RazorLangVersion>3.0</RazorLangVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="ComponentLibWithXmlLinkerConfig.xml" />
```

Comment:
Is this necessary? What happens if you remove this line?

Reply:
Works without it.